impl/etcd/queue/etcd.go

```diff
@@ -145,11 +170,26 @@ func (q *Queue) dequeue(key, value []byte, rev int64, cbs callbacks) error {
 		return err
 	}
 
+	// Read mutation sequence number.
+	bSequence, err := q.srvConfig.Read(q.ctx, txn, sequenceConfigKey)
```

Comment:
I think this is problematic because it creates a single row that's operating as a global sequence counter. That row will get hot. A few ideas:

- Use the MySQL incrementing counter here. 
- Add support for time based ordering with an implicit sequence number (ala spanner)
- Write the sequence number in batches within an epoch. ie. reset the sequence number to 0 at the start of each epoch. The idea is that we should probably write all the mutation that are happening in an epoch together, and we'll know for sure which order we want them in.